main/webapp/modules/core/styles/index/open-project-ui.less

```diff
@@ -111,113 +111,168 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 }
 
 #metadata-body table {
-  border-collapse:collapse
+  border-collapse: collapse;
+  table-layout: fixed;
 }
 
-#metadata-body td {
-    padding: 1px
+table.list-table{
+    margin:0;
 }
 
-#metadata-body tr:nth-child(even) {background: #f2f2f2}
-#metadata-body tr:nth-child(odd) {background: #FFF}
+#metadata-body td {
+  padding: 1px;
+}
```

Comment:
Are those style changes really related to this PR? Could you just explain briefly why you had to do this?

Reply:
After doing the adjustments stated [here](https://github.com/OpenRefine/OpenRefine/pull/4690#issuecomment-1088144300), I thought the table data would look more visually appealing with a little space around them. They seemed too close to the table borders

Reply:
> After doing the adjustments stated https://github.com/OpenRefine/OpenRefine/pull/4690#issuecomment-1088144300, I thought the table data would look more visually appealing with a little space around them. They seemed too close to the table borders

@wetneb this statement is wrong. Actually, that code was a duplicate so I took out one instance of it